control/pid.go

```diff
@@ -22,25 +22,51 @@ func (l *Loop) newPID(config BlockConfig, logger logging.Logger) (Block, error)
 
 // BasicPID is the standard implementation of a PID controller.
 type basicPID struct {
-	mu       sync.Mutex
-	cfg      BlockConfig
-	error    float64
-	kI       float64
-	kD       float64
-	kP       float64
-	int      float64
+	mu     sync.Mutex
+	cfg    BlockConfig
+	logger logging.Logger
+	// used by the single input/output controller
+	error float64
+	kI    float64
+	kD    float64
+	kP    float64
+	int   float64
+	tuner pidTuner
```

Comment:
is this individual tuner for the single input/output case? thinking about if there's an easy way we can use `tuners` for both single IO and MIMO, and in the single case, `tuners` just has one tuner in it

Reply:
yeah this is only for the single input/output case. ideally once mimo is finished it will be removed to only use `tuners`.

I can go ahead and swap to using `tuners` in this pr as well as some other migration changes that could happen now.